File: src/Actors/Empleado.java
package Actors;

public class Empleado extends Usuario{
    
    public Empleado(String nombre, String apellidos, String telefono, String identificacion, String correo, String contrasenia, String rol) {
        super(nombre, apellidos, telefono, identificacion, correo, contrasenia, "Empleado");
    }

}

File: Códigos/Actores/Empleado.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProyectoEstructuras.Códigos.Actores
{
    public class Empleado : Usuario
    {
        public Empleado(string nombre, string apellidos, string telefono, string identificacion, string numeroIdentificacion, string correo, string contrasenia, string rol)
            : base(nombre, apellidos, telefono, identificacion, numeroIdentificacion, correo, contrasenia, "Empleado")
        {
        }
    }
}
